.github/workflows/go-fuzz-test.yml

```
@@ -0,0 +1,13 @@
+name: Go fuzz test
+on:
+  push:
+  pull_request:
+jobs:
+  fuzz-test:
+    name: Fuzz test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jidicula/go-fuzz-action@v1.1.0
```

Nitpick: hash-pin here to prevent tag overwrites:

```suggestion
      - uses: jidicula/go-fuzz-action@4f24eed45b25214f31a9fe035ca68ea2c88c6a13 # v1.2.0
```